src/pages/RestaurantPage/RestaurantPage.spec.js

```diff
@@ -3,20 +3,7 @@ import RestaurantPage from './RestaurantPage'
 
 describe('RestaurantPage', () => {
```

Comment:
You could add another test which tests whether the function `onNavigate` will be called on a click!

Reply:
Done! :D